proxy/proxy.go

```diff
@@ -116,10 +116,16 @@ func NewProxy(c Config) (*Proxy, error) {
 
 func (proxy *Proxy) AttachExistingContainers() {
 	containers, _ := proxy.client.ListContainers(docker.ListContainersOptions{})
-	for _, cont := range containers {
-		if strings.HasPrefix(cont.Command, weaveWaitEntrypoint[0]) {
-			proxy.ContainerStarted(cont.ID)
+	for _, apiContainer := range containers {
+		container, err := proxy.client.InspectContainer(apiContainer.ID)
+		if err != nil {
+			Log.Warningf("unable to attach existing container %s since inspecting it failed: %v", apiContainer.ID, err)
```

Comment:
As I said before, I don't think we should log any warning if the container has simply disappeared.